Makefile

```diff
@@ -122,10 +124,10 @@ clean:
 BOARD =
 
 matek_H7_slim:
-	${MAKE} stm32h7xx BOARD=MATEK_H743_SLIM
-
+	${MAKE} stm32h7xx BOARD=MATEK_H743_SLIM BOARD_FILE_NAME=$@
+	
```

Comment:
Can you please remove extra tab here? Everything else looks good to me.

Reply:
removed, thx!